k8s/spinnaker/scripts/install/setup.sh

```diff
@@ -20,7 +20,7 @@ NUM_ENABLED_APIS=$(gcloud services list --project $PROJECT_ID \
 
 if [ $NUM_ENABLED_APIS != $NUM_REQUIRED_APIS ]; then
   bold "Enabling required APIs ($REQUIRED_APIS)..."
-  bold "This will take a few minutes (progress will not be reported during this operation)..."
+  bold "This may take 10-15 minutes or more (progress will not be reported during this operation)..."
```

Comment:
I'd say "...10 minutes or more..." or "...15 minutes or more..."
Which ever sounds more accurate to you. IMO having a range along with "at least" or "or more" defeats the purpose of the range

Reply:
Updated; thanks!